secant/Wrappers/WrappedSDKSynchronizer.swift

```diff
@@ -89,6 +94,14 @@ class LiveWrappedSDKSynchronizer: WrappedSDKSynchronizer {
             })
             .store(in: &cancellables)
         
+        NotificationCenter.default.publisher(for: .blockProcessorFinished)
+            .compactMap { $0.userInfo?[CompactBlockProcessorNotificationKey.latestScannedBlockHeight] as? BlockHeight }
```

Comment:
uhmm this shouldn't be what we get this piece of info from, but the synchronizer itself

Reply:
you should use this instead

````
 case .synced:
            NotificationCenter.default.post(
                name: Notification.Name.synchronizerSynced,
                object: self,
                userInfo: [
                    SDKSynchronizer.NotificationKeys.blockHeight: self.latestScannedHeight
                ]
            )
````

Reply:
fixed